kraken/kubernetes/client.py

```diff
@@ -388,3 +390,24 @@ def find_kraken_node():
         except Exception as e:
             logging.info("%s" % (e))
             sys.exit(1)
+
+
+# Watch for a specific node status
+def watch_node_status(node, status, timeout, resource_version):
+    for event in watch_resource.stream(
+        cli.list_node,
+        field_selector=f"metadata.name={node}",
+        timeout_seconds=timeout,
+        resource_version=f"{resource_version}",
+    ):
+        conditions = [status for status in event["object"].status.conditions if status.type == "Ready"]
```

Comment:
Need to add in a path for covering when the cluster becomes disconnected like what was added in [PR ](https://github.com/chaos-kubox/krkn/pull/217)